api/src/domain/updateSidebarCards.test.ts

```diff
@@ -120,11 +114,11 @@ describe("updateRoadmapSidebarCards", () => {
       expect(getCurrentBusiness(updatedUserData).preferences.visibleSidebarCards).toContain(formationNudge);
     });
 
-    it("removes formation-nudge if  operatingPhase is NEEDS_TO_REGISTER_FOR_TAXES", () => {
+    it("removes formation-nudge if operatingPhase is FORMED", () => {
```

Comment:
pebble - since only 1 card appears for a phase, I think it's better to have a test like this card renders with this specific phase, And additional test for all the other phases to validate that the card does not appear